boms/cloud-oss-bom/pom.xml

```diff
@@ -7,7 +7,7 @@
 
   <groupId>com.google.cloud</groupId>
   <artifactId>libraries-bom</artifactId>
-  <version>2.7.0-SNAPSHOT</version>
+  <version>2.7.1-SNAPSHOT</version>
```

Comment:
next release is more likely to be 2.8.0 since google-cloud-java increments their minor version almost weekly